source/Integrate/Tutorials/PieSync/PieSync_partner_form.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+weight: 0
+title: Sync your SendGrid contacts with other apps
+navigation:
+  show: true
+seo:
+  title: Sync your SendGrid contacts with other apps
+  description: Learn how to keep your SendGrid contacts in sync with 100+ other apps
+  keywords: integrate, tutorial, contact sync, PieSync, Mailchimp, Zendesk, Salesforce, HubSpot, Quickbooks
+---
+
+Setting up a sync is as easy as following these steps:
+1. [Create a PieSync account](#-Create-a-PieSync-account)
+2. [Select your apps](#Select-your-apps)
+3. [Authorize your apps](#Authorize-your-apps)
+4. [Set up your sync configuration](#Set-up-your-sync-configuration)
+
+{% anchor h2 %}
+Create a PieSync account
+{% endanchor %}
+
+Sign up for a PieSync account [here](https://app.piesync.com/).
+
+{% anchor h2 %}
+Select your apps
+{% endanchor %}
+
+Select Sendgrid and the app you would like to sync with from within the PieSync application.
+
+{% anchor h2 %}
+Authorize your apps
+{% endanchor %}
+
+To authorize, follow the steps as described and make sure you have your [API Key]({{root_url}}/User_Guide/Settings/api_keys.html) before getting started.
+
+** SENDGRID IMAGE 1 **
```

Comment:
Can you please add the images as GitHub comments so we can get them into this doc? Thanks!

Reply:
@Whatthefoxsays the images are in the images folder. Isn't this sufficient?
For instance for image 1: https://github.com/woutdegeyter/docs/blob/cecaab08c2d21ca4e5338be1e63ec985beb8ebb8/source/Integrate/Tutorials/PieSync/images/sendgrid%20image%201.png